Samples/macOS-Swift/macOS-Swift/ViewController.swift

```diff
@@ -14,6 +14,19 @@ class ViewController: NSViewController {
         // Update the view, if already loaded.
         }
     }
+    
+    @IBAction func addBreadCrumb(_ sender: Any) {
+        let crumb = Breadcrumb()
+        crumb.message = "tapped addBreadcrumb"
+        SentrySDK.addBreadcrumb(crumb: crumb)
```

Comment:
L: maybe it'd be useful to call captureExeption or event here too as only the `addBreadcrumb won't report an event along with the breadcrumb, so one needs to click on the 2 buttons and this is not intuitive.
https://github.com/getsentry/sentry-android/blob/master/sentry-sample/src/main/java/io/sentry/sample/MainActivity.java#L38-L47

Reply:
Actually I like it that the button doesn't trigger `captureExeption`. With just adding the breadcrumb, I can add multiple breadcrumbs and then trigger whatever action I want. Like this I am more flexible. Still I added some more context to the breadcrumb.

Reply:
@philipphofmann that's fine, but then you need to be aware that people will open issues, telling that the breadcrumb button doesn't work, as it happened with me ^^